code/go/0dns.io/zdnscore/go.mod

```diff
@@ -2,10 +2,14 @@ module 0dns.io/zdnscore
 
 require (
 	0dns.io/core v0.0.0
-	github.com/0chain/gosdk v1.1.8
-	go.mongodb.org/mongo-driver v1.4.0
-	go.uber.org/zap v1.15.0
-	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
+	github.com/0chain/gosdk v1.2.77-0.20210706112949-6fe9b413de72
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/gedex/inflector v0.0.0-20170307190818-16278e9db813 // indirect
+	github.com/idubinskiy/schematyper v0.0.0-20190118213059-f71b40dac30d // indirect
+	go.uber.org/zap v1.18.1
+	golang.org/x/mobile v0.0.0-20200329125638-4c31acba0007 // indirect
+	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 )
 
 replace 0dns.io/core => ../core
```

Comment:
please merge latest code. I have move go.mod to top folder. and it has only one module 0dns.io instead of 2 modules.